lib/arjdbc/postgresql/base/pgconn.rb

```diff
@@ -1,8 +1,8 @@
-module ActiveRecord::ConnectionAdapters::PostgreSQL::OID
-  class PGconn # emulate PGconn#unescape_bytea due #652
```

Comment:
I can hopefully make a pull request tomorrow to try to keep them scoped to where they are accessed if that would be better. Otherwise, we have to override other logic so that we can bypass the references. The ideal solution would probably be to get some patches into ActiveRecord to make it less PG dependent, but it seems like that would be better to handle when working on 5.1 or 5.2 compatibility, if the Rails team would accept those changes.